### PR TITLE
First commit. with Sign in Page, Auth Guards, Injectors folder structure etc etc and etc

### PR DESCRIPTION
**Pull request by Julias0**

### CODE REVIEW

File: src/app/auth/auth.module.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+
```
Comment (iamtarun1993) [inline src/app/auth/auth.module.ts:4]: remove this extra line

File: src/app/auth/sign-in/sign-in.page.ts [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    await this.loaderService.showLoader();
+
+    const checkEmailExists$ = this.routerAuthService
+      .checkEmailExists(this.fg.controls.email.value)
+      .pipe(
+        catchError(err => {
+          this.handleError(err);
+          return throwError(err);
+        }),
+        shareReplay(),
```
Comment (iamtarun1993) [inline src/app/auth/sign-in/sign-in.page.ts:42]: nice...

File: src/app/core/services/auth.service.ts [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+  constructor(
+    private storageService: StorageService,
+    private tokenService: TokenService,
+    private apiService: ApiService,
+    private dataTransformService: DataTransformService
+  ) { }
+
+  getEou(): Promise<ExtendedOrgUser> {
+    return this.storageService.get('user');
```
Comment (iamtarun1993) [inline src/app/core/services/auth.service.ts:23]: not sure, but we also nee to pull ExtendedOrgUser from api if user is not present in storageService

Comment (Julias0) [reply src/app/core/services/auth.service.ts:23]: This will always be present since we set this when the token is set. Same as out web app / mobile app

**Pull request merged**